templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -6,142 +6,341 @@
+    "|Llama-3.1| `meta-llama/Meta-Llama-3.1-8B-Instruct` | 4096 |\n",
+    "|Llama-3.1| `meta-llama/Meta-Llama-3.1-70B-Instruct`  | 4096 |\n",
+    "|Llama-3| `meta-llama/Meta-Llama-3-8B-Instruct` | 4096 |\n",
+    "|Llama-3| `meta-llama/Meta-Llama-3-70B-Instruct`| 4096 |\n",
+    "|Mistral| `mistralai/Mistral-Nemo-Instruct-2407`  | 4096 |\n",
+    "|Mistral| `mistralai/Mistral-7B-Instruct-v0.3` | 4096 |\n",
+    "|Mixtral| `mistralai/Mixtral-8x7B-Instruct-v0.1` | 4096 |\n",
     "\n",
-    "# Launch a LoRA fine-tuning job for Llama 3 8B with 16 A10s\n",
-    "!llmforge anyscale finetune training_configs/lora/llama-3-8b.yaml\n",
     "\n",
-    "# Launch a full-param fine-tuning job for Llama 3 8B with 16 A10s\n",
-    "# !llmforge anyscale finetune  training_configs/full_param/llama-3-8b.yaml"
+    "Note: \n",
+    "- Cluster type for all models: 8xA100-80G\n",
+    "- Supported context length for models: 512 up to max. context length of each model in powers of 2."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`LLMForge` is an Anyscale CLI and library that is installed on this workspace so that you can quickly experiment and customize various LLM finetuning experiments by simply modifying a config file. For extensive documentation around what is supported through the config refer to [docs](https://docs.anyscale.com/reference/finetuning-config-api/). "
+    "## Summary of Features in Custom Mode\n",
+    "\n",
+    "### ✅ Support both Full parameter and LoRA\n",
+    "\n",
+    "* LoRA with different configurations, ranks, layers, etc. (Anything supported by huggingface transformers)\n",
+    "* Full-parameter with multi-node training support\n",
+    "    \n",
+    "### ✅ State of the art performance related features:\n",
+    "\n",
+    "* Gradient checkpointing\n",
+    "* Mixed precision training\n",
+    "* Flash attention v2\n",
+    "* Deepspeed support (zero-DDP sharding)\n",
+    "\n",
+    "### ✅ Unified chat data format with flexible prompt format support enabling finetuning for:\n",
+    "\n",
+    "\n",
+    "#### Use-case: Multi-turn chat, Instruction tuning, Classification:\n",
+    "\n",
+    "Example data format (JSON):\n",
+    "```json\n",
+    "{\n",
+    "    \"messages\": [\n",
+    "        {\"role\": \"system\", \"content\": \"You are a helpful assistant.\"},\n",
+    "        {\"role\": \"user\", \"content\": \"Hi\"},\n",
+    "        {\"role\": \"assistant\", \"content\": \"Howdy!\"},\n",
+    "        {\"role\": \"user\", \"content\": \"What is the type of this model?\"},\n",
+    "        {\"role\": \"assistant\", \"content\": \"[[1]]\"},\n",
+    "    ]\n",
+    "}\n",
+    "```\n",
+    "\n",
+    "Prompt Format for llama-3-instruct (YAML):\n",
+    "\n",
+    "```yaml\n",
+    "system: \"<|start_header_id|>system<|end_header_id|>\\n\\n{instruction}<|eot_id|>\"\n",
+    "user: \"<|start_header_id|>user<|end_header_id|>\\n\\n{instruction}<|eot_id|>\"\n",
```

Comment:
FYI we need a small llmforge change for this prompt format to be supported. `trailing_assistant` is currently a required field.  (i'll take it up soon)

Reply:
It's ok I didn't mean for this to be complete. It's just meant to be part of an example.